README.md

```diff
@@ -128,7 +128,7 @@ Optionally, you may name the return value, similar to named parameters:
 Or even name a signature to reuse it later on:
 
 ```js
-connect(options: Object) => connection: Object
+connect = (options: Object) => connection: Object
```

Comment:
If we had tools depending on this README that would be a breaking change; I think those changes should be proposed on a separate PR.